packages/sui-precommit/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@schibstedspain/sui-precommit",
+  "version": "1.1.0",
+  "description": "Installs a git hook that executes sui commit rules",
+  "main": "index.js",
+  "scripts": {
+    "build": "echo \"building...\"",
+    "install": "./install.js"
```

Comment:
That's being fixed on the actual pre-commit-rules for Windows.

`"install": "node ./install.js"`